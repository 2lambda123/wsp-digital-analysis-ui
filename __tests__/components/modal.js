/* global describe, it, expect, jest */

import React from 'react'
import renderer from 'react-test-renderer'

jest.mock('react-modal', () => 'React-Modal')

import Modal from '../../lib/components/modal'

describe('Component > Modal', () => {
  it('renders correctly', () => {
    const onRequestCloseFn = jest.fn()
    const tree = renderer.create(
      <Modal onRequestClose={onRequestCloseFn}>
        Modal content
      </Modal>
    ).toJSON()
    expect(tree).toMatchSnapshot()
    expect(onRequestCloseFn).not.toBeCalled()
  })
})
