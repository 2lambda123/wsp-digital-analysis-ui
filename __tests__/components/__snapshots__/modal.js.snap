exports[`Modal renders correctly 1`] = `
<React-Modal
  isOpen={true}
  onRequestClose={[Function mockConstructor]}
  shouldCloseOnOverlayClick={true}
  style={
    Object {
      "overlay": Object {
        "zIndex": 2500
      }
    }
  }>
  Modal content
</React-Modal>
`;
